examples/web/index.html

```diff
@@ -64,7 +64,7 @@
         padding: 0.25rem;
       }
 
-      .example-TextField-grid {
+      .example-TextField-grid, .example-CheckboxField-grid {
```

Comment:
Can you put this class on new line? It improves readability.

Reply:
Fixed.